settings.py

```diff
@@ -15,6 +15,10 @@
     }
 }
 
+TIMEZONE = "Ukraine/Kiev"
 USE_TZ = False
 
 INSTALLED_APPS = ("db",)
+
+# ALLOWED_HOSTS = ['localhost', '127.0.0.1', '[::1]']
+DEBUG = True
```

Comment:
The `DEBUG` setting is set to `True`. Ensure this is turned off (`DEBUG = False`) in a production environment to avoid exposing sensitive information.